examples/ffi/README.md

```diff
@@ -0,0 +1,23 @@
+
+# Build FFI examples
+*All the commands in this README are executed at the root level.*
+
+In order to be able to run FFI examples, you need to build the shared library and headers with the command:
+```c
+// (dont worry too much about the fact that this is named "profiling")
+./build-profiling-ffi.sh ./build
```

Comment:
alternatively: 
```
cargo run --bin release --features profiling,telemetry,data-pipeline,symbolizer,crashtracker --release -- --out
```

Reply:
+1, we're working towards removing the scripts so it would be good that people running the examples use the new way to build the project.